### PR TITLE
[KEYCLOAK-7264] Add documentation about the new RoleMappingsProvider …

### PR DESCRIPTION
**Pull request by sguilhen**
…SPI.

Link to Jira: https://issues.jboss.org/browse/KEYCLOAK-7264

### CODE REVIEW

File: securing_apps/topics/saml/java/general-config/sp_role_mappings_provider_element.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+The configuration of the provider looks as follows:
+
+[source,xml]
+----
+...
+<RoleIdentifiers>
+    ...
+</RoleIdentifiers>
+<RoleMappingsProvider id="properties-based-role-mapper">
+    <Property name="properties.file.location" value="/WEB-INF/role-mappings.properties"/>
```
Comment (mhajas) [inline securing_apps/topics/saml/java/general-config/sp_role_mappings_provider_element.adoc]:
```suggestion
    <Property name="properties.resource.location" value="/WEB-INF/role-mappings.properties"/>
```

**Review by mhajas — COMMENTED** [review]
Looks good to me, I have just minor comment on one of the examples.

**Comment by mhajas** [comment]
> Looks good to me, I have just minor comment on one of the examples.

File: securing_apps/topics/saml/java/general-config/sp_role_mappings_provider_element.adoc (hunk 1 above)
Comment (sguilhen) [reply securing_apps/topics/saml/java/general-config/sp_role_mappings_provider_element.adoc]: Good catch! Fixed

**Review by mhajas — DISMISSED** [review]

File: upgrading/topics/keycloak/changes.adoc [hunk 2]
```diff
@@ -1,5 +1,20 @@
 == Migration Changes
 
+=== Migrating to 8.0.0
```
Comment (hmlnarik) [inline upgrading/topics/keycloak/changes.adoc]: Is there potential to affect current deployments and any migration needed? If not, this should be placed rather into release_notes/topics/8_0_0.adoc

File: securing_apps/topics.adoc [hunk 3]
```diff
@@ -70,6 +70,9 @@ include::topics/saml/java/general-config/sp-keys/keystore_element.adoc[]
 include::topics/saml/java/general-config/sp-keys/key_pems.adoc[]
 include::topics/saml/java/general-config/sp_principalname_mapping_element.adoc[]
 include::topics/saml/java/general-config/roleidentifiers_element.adoc[]
+ifeval::[{project_community}==true]
```
Comment (hmlnarik) [inline securing_apps/topics.adoc]: This condition is not necessary

File: upgrading/topics/keycloak/changes.adoc (hunk 2 above)
Comment (sguilhen) [reply upgrading/topics/keycloak/changes.adoc]: Done!

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**